### ISSUE TITLE
yo buddy,How can I run this program?

### ISSUE THREAD

**Issue by RockyFu267** [hourliesss/GolangPoker#1]
D:\workspace\src\GolangPoker>go run main.go
# command-line-arguments
.\main.go:13:7: undefined: launchGame
.\main.go:15:11: undefined: coefficients
.\main.go:28:3: undefined: showCards
.\main.go:34:4: undefined: play
.\main.go:72:5: undefined: play
.\main.go:96:16: undefined: person
.\main.go:109:17: undefined: coefficients
.\main.go:119:17: undefined: coefficients
.\main.go:128:8: undefined: newGame
.\main.go:133:4: undefined: showCards
.\main.go:133:4: too many errors

I didn't find the cause of the problem

**Comment by RockyFu267** [comment]
BTW,I'm also writing my first golang program.Texas hold 'em too.
I wrote the shuffle, deal, compare the size.
But I I'm not going to write the logic of the legitimacy of the actions of multiple people.
I don't know how to write a multiplayer game.

**Comment by hourliesss** [comment]
Hi Rocky, I wrote the logic to play multiplayer but it is pretty basic as
we only play on a single terminal.

I tried to had bots to play with on this version so there are many files to
include.
You can try to play with bots but they are easy to defeat. I will improve
those when I have the time.

To launch the game:

*go run main.go cards.go game.go odds.go player.go math_functions.go*

Tell me if it works, I am not currently working on this project but I
available to help you if you had any questions!

Le ven. 27 déc. 2019 à 21:23, RockyFu267 <notifications@github.com> a
écrit :

**Comment by RockyFu267** [comment]
D:\workspace\src\GolangPoker>go run main.go cards.go game.go odds.go player.go math_functions.go
# command-line-arguments
.\main.go:15:11: undefined: coefficients
.\main.go:109:17: undefined: coefficients
.\main.go:119:17: undefined: coefficients
.\game.go:352:79: undefined: coefficients


I think that's the problem
`type coefficients struct {
	normalisation int
	check         float64
	raise         float64
	fold1         float64
	fold2         float64
	call          float64
	allIn         float64
}`
this code is in bot.go.
so mv it to another file. It works.

**Comment by hourliesss** [comment]
Yes you should include bot.go and not odds.go (useless), sorry for the
mistake

Le lun. 30 déc. 2019 à 20:43, RockyFu267 <notifications@github.com> a
écrit :
